### ISSUE TITLE
Throwing error while attempting to open Excel files

### ISSUE THREAD

**Issue by dileeplj** [open-rpa/openrpa#403]
When I'am trying to open an Excel/CSV file using The "Read Range" or "Read Cell" function in OpenRPA, I am getting the error as in the attached image. 

The same workflow used to function flawlessly, the issue has come up only after a recent Windows update.
![Capture](https://user-images.githubusercontent.com/71010456/92586832-69209700-f2b4-11ea-815e-0ffeef1fad6e.PNG)

**Comment by skadefro** [comment]
Please check the latest version of the robot and see if that fixes the error

**Comment by dileeplj** [comment]
Thank you for the resolution. I will try this out.

**Comment by skadefro** [comment]
Did it work ?

**Comment by skadefro** [comment]
I'm going to assume it worked, else feel free to request for re-opening this issue

**Comment by dileeplj** [comment]
Extremely sorry for the late response. I did not see your previous comments. 

I have installed the updated version of OpenRPA (Downloaded on 25/9/2020 - 1:30 PM IST)

Still the same error is shown when I used "ReadCell"

Belw is a screenshot of the error.

![image](https://user-images.githubusercontent.com/71010456/94392385-c00fe280-0175-11eb-8822-968aaf022b63.png)

Here is the version details.

![image](https://user-images.githubusercontent.com/71010456/94392493-0bc28c00-0176-11eb-9707-0bdd7e38fda2.png)
